crates/rover-client/src/query/subgraph/check.graphql

```diff
@@ -17,6 +17,11 @@
         compositionValidationResult {
           errors {
             message
+            locations {
```

Comment:
I can't speak to this part but the rest lgtm 😄

Reply:
OOPS

Reply:
fixed - good catch 😄